config/helpers.php

```diff
@@ -876,23 +878,23 @@ function twitter(string $username, string $text = null, string $title = null, st
 /**
  * Shortcut for url()
  *
- * @param string $path
+ * @param string|null $path
  * @param array|string|null $options
  * @return string
  */
-function u(string $path = null, $options = null): string
+function u(?string $path = null, $options = null): string
```

Comment:
Why not? …

```suggestion
function u(?string $path = null, ?$options = null): string
```

Reply:
Is this valid syntax, so without type?

Reply:
Oh wow,  you are completely right. Of course not. Forget my comment :)